_overviews/core/nightlies.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: singlepage-overview
+title: Nightly Versions of Scala
+permalink: /overviews/core/:title.html
+---
+
+We regularly publish nightly versions of both Scala 2 and 3 so that users can preview and test the contents of upcoming releases.
+
+We informally call them "nightly" versions, but technically it's a misnomer. A so-called “nightly” is built for every merged PR.
+
+Here's how to find and use these versions.
+
+## Scala 3
+
+Scala 3 nightly versions are published to Maven Central. If you know the full version number of the nightly you want to use, you can use it just like any other Scala 3 version.
+
+One quick way to get that version number is to visit https://dotty.epfl.ch and look in the upper left corner.
+
+Another way is to scrape Maven Central, as shown in this script: https://raw.githubusercontent.com/VirtusLab/community-build3/master/scripts/lastVersionNightly.sc
+
+A third way is to use [scala-cli](https://scala-cli.virtuslab.org), as follows.
+
+### scala-cli
+
+You can run nightlies with commands such as:
+
+    scala-cli -S 3.nightly
+    scala-cli -S 3.3.nightly
+
+The default command is `repl`, but all the other scala-cli subcommands such as `compile` and `run` work, too. It also works with `//>` directives in your script itself, for example:
+
+    //> using scala 3.nightly
+
+See this [scala-cli doc page](https://scala-cli.virtuslab.org/docs/commands/compile#scala-nightlies) for details.
+
+## Scala 2.13 or 2.12
+
+Scala 3 nightly versions are published to a special resolver. Unless you are using scala-cli, you'll need to add that resolver to your build configuration in order to use these versions.
+
+### quick version (sbt)
+
+    Global / resolvers += "scala-integration" at
+      "https://scala-ci.typesafe.com/artifactory/scala-integration/"
+    scalaVersion := "2.13.15-bin-abcd123"
+
+for a 2.12 nightly, substitute e.g. `2.12.20` for `2.13.15`; in either case, it's the version number of the _next_ release on that branch
+
+for `abcd123`, manually substitute the first 7 characters of the SHA of the latest green build [on the 2.13.x or 2.12.x branch on Travis-CI](https://app.travis-ci.com/github/scala/scala/branches).
```

Comment:
what I usually do is look at https://github.com/scala/scala/commits/2.13.x, click on the CI icon for the merge commit. then the version shows up in the scala-dist travis task:

<img width="1121" alt="image" src="https://github.com/user-attachments/assets/d5681a6c-da3e-447d-b56f-39305c70fd39">